scripts/check-sse4_2.sh

```diff
@@ -44,7 +44,9 @@ for dir in $dirs; do
             # f2.*0f 38 is the opcode of `crc32`, see Intel® SSE4 Programming Reference
             found=0
             for sym in $fast_crc32; do
-                if [[ `objdump --disassemble="$sym" $dirfile 2> /dev/null | grep ".*f2.*0f 38.*crc32"` ]]; then
+		read -r start stop <<<$(nm "$dirfile" | grep -A1 "$sym" | awk '{printf("0x"$1" ")}')
```

Comment:
I think this expression is not quite correct. The `grep -A1` is picking the address of whatever symbol is _alphabetically_ after Fast_CRC32, and treating it like it is the address of the next symbol by-address.

If you call `nm` as `nm -n` it will print sorted by address, and - bonus! - it runs much faster.

Reply:
What does the `<<<` redirection do? It's hard to google for.

Reply:
> If you call `nm` as `nm -n` it will print sorted by address, and - bonus! - it runs much faster

Thanks a lot! I will fix this.

> What does the `<<<` redirection do? It's hard to google for.

It is a here string. See https://renenyffenegger.ch/notes/Linux/shell/bash/built-in/read for more details.